metadata.rb

```diff
@@ -18,6 +18,9 @@
 depends 'rightscale_backup', '~> 1.2.0'
 depends 'dns', '~> 0.1.3'
 depends 'git', '~> 4.0.2'
+depends 'aws', '~> 2.9.3'
```

Comment:
I think this may need to go into kabam-rs-mysql not rs-mysql.  I don't think rs-mysql has any dependency on aws.